cpp/mail_krimdok_newsletter.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -o errexit -o nounset
+
+
+BACKUP_DIR=/usr/local/var/lib/tuelib/sent_newsletters
+mkdir --parents "$BACKUP_DIR"
+REPLY_TO=johannes.ruscheinski@uni-tuebingen.de
+SQL_QUERY="SELECT 'johannes.ruscheinski@uni-tuebingen.de'"
+declare -i count=0
+for regular_file in $(ls -p | grep -v /); do
+    bulk_mailer "$regular_file" \
+                "$SQL_QUERY" \
```

Comment:
SQL query should be escaped before using it as arg (might contain double quotes, backslashes and so on)

Reply:
Inlined the SQL statement.